Chutzpah/TestRunner.cs

```diff
@@ -684,8 +684,8 @@ private static string BuildRunnerArgs(TestOptions options, TestContext context,
             string runnerArgs;
             var testModeStr = testExecutionMode.ToString().ToLowerInvariant();
             var timeout = context.TestFileSettings.TestFileTimeout ?? options.TestFileTimeoutMilliseconds ?? Constants.DefaultTestFileTimeout;
-
-            runnerArgs = string.Format("--ignore-ssl-errors=true --proxy-type=none --ssl-protocol=any \"{0}\" {1} {2} {3} {4} {5}",
+            runnerArgs = string.Format("--ignore-ssl-errors=true {0} --ssl-protocol=any \"{1}\" {2} {3} {4} {5} {6}",
+                                       proxySetting,

```

Comment:
The CI looks like it is still failing: https://ci.appveyor.com/project/mmanela/chutzpah/build/4.3.0-73